Makefile

```
@@ -317,12 +331,21 @@ setup-dev: ##@setup Install all necessary tools for development
 setup-dev:
 	echo "Replaced by Nix shell. Use 'make shell' or just any target as-is."
 
+USE_SYSTEM_NIM ?= 0
+
+generate-sds:  ##@ Build libsds third_party
+	cd vendor/github.com/waku-org/sds-go-bindings/sds/ && make build USE_SYSTEM_NIM=$(USE_SYSTEM_NIM)
```

Why not just use `-C` flag?

```suggestion
	$(MAKE)  -C vendor/github.com/waku-org/sds-go-bindings/sds build USE_SYSTEM_NIM=$(USE_SYSTEM_NIM)
```

---

Same below.

---

Thanks! Done in https://github.com/status-im/status-go/pull/6884/commits/bd2d740b6c3fc92a049da3563f0e135dfe44eccc

EDITED: I will revert this change because it causes the following issue in Nix:
```nim
status-go-e3e91272607ab0b455be2f6ea719d7df25280208-dirty> Running 'make update'...
status-go-e3e91272607ab0b455be2f6ea719d7df25280208-dirty> make[2]: Entering directory '/build/status-go-library/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds'
status-go-e3e91272607ab0b455be2f6ea719d7df25280208-dirty> make[2]: *** No rule to make target 'update'.  Stop.
status-go-e3e91272607ab0b455be2f6ea719d7df25280208-dirty> make[2]: Leaving directory '/build/status-go-library/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds'
status-go-e3e91272607ab0b455be2f6ea719d7df25280208-dirty> make[1]: *** [Makefile:19: prepare] Error 2
status-go-e3e91272607ab0b455be2f6ea719d7df25280208-dirty> make[1]: Leaving directory '/build/status-go-library/vendor/github.com/waku-org/sds-go-bindings/sds'
status-go-e3e91272607ab0b455be2f6ea719d7df25280208-dirty> make: *** [Makefile:337: generate-sds] Error 2
error: builder for '/nix/store/490rrwfcw9lqkicc2giymi0k3d41s140-status-go-e3e91272607ab0b455be2f6ea719d7df25280208-dirty.drv' failed with exit code 2;
       last 10 log lines:
       > e3e91272607ab0b455be2f6ea719d7df25280208-dirty
       > e3e91272607ab0b455be2f6ea719d7df25280208-dirty
       > e3e91272607ab0b455be2f6ea719d7df25280208-dirty
       > Running 'make update'...
       > make[2]: Entering directory '/build/status-go-library/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds'
       > make[2]: *** No rule to make target 'update'.  Stop.
       > make[2]: Leaving directory '/build/status-go-library/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds'
       > make[1]: *** [Makefile:19: prepare] Error 2
       > make[1]: Leaving directory '/build/status-go-library/vendor/github.com/waku-org/sds-go-bindings/sds'
       > make: *** [Makefile:337: generate-sds] Error 2
       For full logs, run 'nix-store -l /nix/store/490rrwfcw9lqkicc2giymi0k3d41s140-status-go-e3e91272607ab0b455be2f6ea719d7df25280208-dirty.drv'.

```